src/main/java/com/sourcegraph/javagraph/MavenProject.java

```diff
@@ -1,6 +1,8 @@
 package com.sourcegraph.javagraph;
 
 import com.google.common.collect.Iterators;
+import com.sourcegraph.javagraph.maven.vendored.*;
+import com.sourcegraph.javagraph.maven.vendored.DefaultModelBuilderFactory;
```

Comment:
we will use the vendored `DefaultModelBuilderFactory` in favor of the apache one